.github/workflows/build-project-linux.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           pip3 install pyaudio
           pip3 install psutil
           pip3 install crcengine
-          pip3 install 'pyinstaller<5.1'
```

Comment:
We should keep this in mind, maybe adding a comment with this and a small description. There will be some version problems possibly somewhen in future ( not only with pyinstaller). So we already have a working solution of limiting specific module versions

Reply:
It is common practice to anchor / freeze the dependency versions to ones known to work. While we're in Alpha / Beta it's probably acceptable to float, but once we hit 'production' the versions should be frozen to get consistent builds.

Reply:
Yes, good point and I agree! We should talk somewhen the next days about a roadmap and a possible first beta release.

Reply:
We may want to migrate to using [`poetry`](https://python-poetry.org/) over `pip` just because it handles that more gracefully, in my opinion. Project for a future time.